nursery/modify-api-blacklist-or-denylist-via-jni-on-android.yml

```diff
@@ -0,0 +1,17 @@
+rule:
+  meta:
+    name: modify API blacklist or denylist via JNI on Android
+    namespace: host-interaction/bypass
+    authors:
+      - mehunhoff@google.com
+    scopes:
+      static: function
+      dynamic: thread
+  features:
+    - and:
+      - os: android
+      - string: "com/android/internal/os/ZygoteInit"
+      - or:
+        - string: "setApiBlacklistExemptions"
+        - string: "setApiDenylistExemptions"
+      - string: "java/lang/String"
```

Comment:
is this string required/useful here?

Reply:
It's used as part of the JNI invocation - I added it to knock down the FP potential but it'd probaly be fine w/out. Thoughts?

Reply:
let's leave it and maybe add a comment